src/Sarif.Multitool.Library/MergeCommand.cs

```diff
@@ -273,7 +273,7 @@ private async Task<bool> FindFilesAsync()
 
                 if (directory.Length == 0)
                 {
-                    directory = @".\";
+                    directory = Directory.GetCurrentDirectory();
                 }
```

Comment:
just a quick update that 
will update the azure pipeline and add ubuntu-latest for this pr to see how the tests run in Linux.

Reply:
able to add azure pipeline to also run test in Linux 
but I found it does not even build before run unit test, 
I think the Powershell script we have have the syntax for window only, I will check and fix issues.

Reply:
fixed, also fixed the build in macos so tests can see how the test run in macos as well